docs/source/install/monitor_without_docker.rst

```diff
@@ -10,7 +10,7 @@ Please note, Scylla recommends you use the Docker version as it will provide you
 The main item to set an alert on is the available disk space in the monitoring system. Data is indefinitely accrued on the Prometheus data directory.
 The current monitoring solution does not churn data.
 
-.. note:: Confirm before installing, that your Grafana and Prometheus versions are supported by the Scylla Monitoring Stack version you want to install. See the `Scylla Monitoring Stack Compatibility Matrix </install/monitoring_stack/#prerequisites>`_.
+.. note:: Confirm before installing, that your Grafana and Prometheus versions are supported by the Scylla Monitoring Stack version you want to install. See the :ref:`Scylla Monitoring Stack Compatibility Matrix <monitoring-matrix>`.
```

Comment:
Look like this is already fixed in https://github.com/scylladb/scylla-monitoring/pull/1438/files